docs/articles/documentation/test-api/assertions/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: docs
+title: Assertions
+permalink: /documentation/test-api/assertions/
+checked: false
+---
+# Assertions
+
+To check if the state of the tested webpage matches the one you expect to see, use *assertions*.
+
+We will provide a comprehensive [set of assertions](assertion-api.md) and all of them based on Behavior Driven Development style (BDD-style).
+The purpose of BDD style is writing tests that reflect the behavior and identify appropriate tests to writing.
+
+This topic contains the following sections.
+
+* [Assertion timeout](#assertion-timeout)
+* [Assertion options](#assertion-options)
+  * [options.timeout](#optionstimeout)
+
+The following example shows how to check the two values for equality:
```

Comment:
The following example demonstrates how to check whether two values are equal or not.